kvm/v12.1.0/README.md

```diff
@@ -0,0 +1,35 @@
+## :zap:  Giant Swarm Release 12.1.0 for KVM :zap:
+
+**This release includes two significant improvements to NGINX Ingress Controller. It also adds support for image registry mirrors, improving reliability.**
+
+**The two NGINX Ingress Controller improvements:**
+* First, multiple NGINX Ingress Controllers per tenant cluster are now supported, enabling separation of internal vs external traffic, dev vs prod, and so on.
+* Second, management of NGINX IC NodePort Service is moved from kvm-operator to NGINX IC App itself. This enables configurability of external traffic policy and lays the foundation for making NGINX IC App optional and not pre-installed in a future KVM platform release.
+- NGINX IC NodePort Service external traffic policy has been made configurable.
+- Management of NGINX IC NodePort Service is moved from kvm-operator to NGINX IC App itself to:
+  - Enable external traffic policy configurability, in a way consistent with other NGINX IC configuration options.
+  - Lay the foundation for making NGINX IC App optional and not pre-installed in a future KVM platform release.
+
+**To change the ownership, `nginx-ingress-controller` NodePort Service in `kube-system` namespace needs to be manually deleted. It then gets to be recreated by triggering the cluster upgrade. To minimize the downtime, please delegate cluster upgrades to your SE.**
```

Comment:
```suggestion
**Note for cluster upgrades:**

Please manually delete `nginx-ingress-controller` NodePort Service in `kube-system` namespace. Upgrading the cluster then recreates the NodePort Service and moves its management from kvm-operator to NGINX IC. To minimize downtime, please delegate cluster upgrades to your SE.**
```

@sslavic 
1. is this a required step? Or is it optional? 
1a If it is required (this is my impression), then I have suggestion in release notes. Pls check if makes it clearer.
1b If it's optional, then the instructions might be better placed in docs.
2. Also, this note applies to cluster upgrades only right? Would it be helpful to make that clearer?